teammapper-frontend/mmp/src/map/handlers/draw.ts

```diff
@@ -265,29 +282,59 @@ export default class Draw {
      */
     public setLink(node: Node) {
         let domLink = node.getLinkDOM()
+        const domText = document.createElementNS('http://www.w3.org/2000/svg', 'text')
```

Comment:
why was this change needed? On first glance, if the link Dom is already present, it makes no sense to draw on top of it?

Reply:
Was testing to see if it made a difference in rendering, I should be able to revert the changes to the method since the issue was with the querySelector and not the setLink() method.